meta/language.LL1

```diff
@@ -0,0 +1,178 @@
+// About this document
+// This is a EBNF *like* description of the Jakt Programming Language
+//
+// EBNF/LL1 dialect:
+// * Blocks are defined by an identifier and an equals, followed by the definition
+// * Literal Characters are surrounded by quotation-marks, preferably double-quotes
+// * Regex-literals matches are surrounded by forward-slashes
+// * Optional groups are surrounded by square-brackets
+// * Repeated groups are surrounded by curly-braces
+// * Groupings are surrounded by round-parentheses
+// * Comments are denoted by a double-slash in-front of them
+// * Multiple possible matches are separated by a vertical-bar/or-sign
+//
+// The most outer scope checks for `Declaration`s
+//
+// This is not yet as fine grained as it could be and does not differentiate
+// between some groups, so this is not (yet) a full guide to tokenization
+// Also the order of definitions and notation is currently not fully consistent
+
+// Stolen from https://webidl.spec.whatwg.org/#prod-i and adjusted to match Jakt more closely
+integer     = /-?([1-9][0-9]*|0[Xx][0-9A-Fa-f]+|0[Oo][0-7]*|0)([ui](8|16|32|64)|uz)?/
+float       = /-?(([0-9]+\.[0-9]*|[0-9]*\.[0-9]+)([Ee][+-]?[0-9]+)?|[0-9]+[Ee][+-]?[0-9]+)(f64)?/
+            | /-?([1-9][0-9]*|0[Xx][0-9A-Fa-f]+|0[Oo][0-7]*|0)f64/
```

Comment:
f32 is also a suffix: https://github.com/SerenityOS/jakt/blob/a0fd770771c134149f8b11605d295280207eb519/src/lexer.rs#L583

I think you asked about `z`, and that's not listed. (I thought it was one but I don't have the best memory.)

Reply:
Thanks
Added that aswell